### PR TITLE
Added File.get_path and File.get_path_absolute functions

### PR DESCRIPTION
**Pull request by marcelofg55**
These are handy when working with 3rd party libraries such as the SteamAPI.

### CODE REVIEW

File: drivers/unix/file_access_unix.h [hunk 1]
```diff
@@ -61,6 +62,9 @@ class FileAccessUnix : public FileAccess {
 	virtual void close(); ///< close a file
 	virtual bool is_open() const; ///< true when file is open
 
+	virtual String get_path() const; /// returns the path for the current open file
```
Comment (akien-mga) [inline drivers/unix/file_access_unix.h]: Shouldn't those be implemented in `FileAccessWindows` too?

Comment (marcelofg55) [reply drivers/unix/file_access_unix.h]: True, I'll add them

**Comment by marcelofg55** [comment]
@akien-mga Just added the windows code too

**Comment by hpvb** [comment]
Thanks!

**Comment by hpvb** [comment]
Cherry picked into 3.0.3

**Pull request merged**